Sources/Bitcoin/BitcoinScript.swift

```diff
@@ -223,7 +223,7 @@ public final class BitcoinScript: BinaryEncoding, CustomDebugStringConvertible {
         var data = Data(capacity: 5 + pubKeyHash.count)
         data.append(contentsOf: [OpCode.OP_DUP, OpCode.OP_HASH160])
         data.append(UInt8(pubKeyHash.count))
-        data.append(pubKeyHash)
+        data.append([UInt8](pubKeyHash), count: pubKeyHash.count)
```

Comment:
I don't understand why this fixes anything 🤔

Reply:
Swift mystery. 

Remember we had similar bug? Data wasn’t converted properly

Reply:
Right